ios/RNGoogleCast/types/RCTConvert+GCKDevice.m

```diff
@@ -27,6 +27,31 @@ + (nonnull id)fromGCKDevice:(nullable GCKDevice *)device {
 
   json[@"modelName"] = device.modelName;
 
+  //Capabilities
+  NSMutableArray<NSString *> *capabilities = [NSMutableArray array];
+  if ([device hasCapabilities:GCKDeviceCapabilityVideoOut]) {
+    [capabilities addObject:@"VideoOut"];
```

Comment:
shouldn't be `addString`?

```suggestion
    [capabilities addString:@"VideoOut"];
```

Reply:
I don't think that's an NSArray selector -
<img width="660" alt="Screen Shot 2022-03-04 at 12 52 25 PM" src="https://user-images.githubusercontent.com/177857/156839934-196f78a5-65d8-4b72-83b4-dc8e2244f8d1.png">

Add object seems to work.  I'm seeing the capabilities coming through on the RN side

Reply:
Yeah I must've been thinking about a `Map`.